charts/nublado/templates/jupyterhub/deployment.template.yml

```diff
@@ -216,3 +227,17 @@ spec:
             - key: jupyterhub_config.py
               path: jupyterhub_config.py
             name: {{ template "nublado.fullname" . }}-jupyterhub-config
+        - name: base-passwd
+          configMap:
+            defaultMode: 420
```

Comment:
Since this is YAML, not JSON, can you use `0644` here and below?

Reply:
I dunno.  If there's a Python parser in the way I need to make it 0o644, and it turns into `420` when you read it back with kubectl in any event.  I can go give it a shot and find out.